examples/accounts_and_transactions.exs

```diff
@@ -0,0 +1,47 @@
+defmodule AccountsAndTransactions do
+  use Application
```

Comment:
`.exs` files will just execute top-to-bottom, so I think you can remove the `defmodule` here and the `def start(_type, _args)`. Then just start with the `IO.puts` as the beginning of the script